tests/test-stbt-auto-selftest.sh

```diff
@@ -0,0 +1,59 @@
+test_auto_selftest_generate()
+{
+    cp -R "$testdir/auto-selftest-example-test-pack" pristine &&
+    cp -R "pristine" "regenerated" &&
+    rm -rf regenerated/selftest/auto_selftest &&
+    cd regenerated &&
+    stbt auto-selftest generate &&
+    cd .. &&
+
+    find . -name '*.pyc' -delete &&
+    diff -ur "pristine" "regenerated"
+}
+
+test_that_generated_auto_selftests_pass_as_doctests()
+{
+    PYTHONPATH=$srcdir python -m doctest \
+        "$testdir/auto-selftest-example-test-pack/selftest/auto_selftest/tests/example_selftest.py"
+}
+
+cd_example_testpack()
+{
+    cp -R "$testdir/auto-selftest-example-test-pack" test-pack &&
+    cd test-pack || fail "Test setup failed"
+}
+
+
+test_that_generated_auto_selftests_pass_stbt_auto_selftest_validate()
+{
+    cd_example_testpack &&
+    stbt auto-selftest validate
+}
+
+test_that_no_auto_selftests_fail_stbt_auto_selftest_validate()
+{
+    cd_example_testpack &&
+    rm -r selftest/auto_selftest &&
+    ! stbt auto-selftest validate
```

Comment:
Is this really the behaviour we want? If we add a `stbt check` that automatically runs pylint + `stbt auto-selftest validate` on every commit, we wouldn't want it to fail for test-packs that haven't defined any FrameObjects or selftest-screenshots.